src/shared-components/bulkErrors/index.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { TransitionGroup } from 'react-transition-group';
+
+import { CenteredErrorList, ErrorList, ErrorItem } from './style';
+import HelperTransition from '../field/helperTransition';
```

Comment:
maybe move this guy into its own directory? might share this with other components down the line. not necessary tho.

Reply:
it is technically shared among 2 components: Bulk and Field. it wasnt copupled in any way to Field so I moved to utils folder. good advice